vendor/github.com/giantswarm/k8scloudconfig/templates.go

```diff
@@ -514,8 +516,8 @@ coreos:
     content: |
       [Unit]
       Description=etcd2
-      Requires=k8s-setup-network-env.service
-      After=k8s-setup-network-env.service
+      Requires=k8s-setup-network-env.service decrypt-tls-assets.service
+      After=k8s-setup-network-env.service decrypt-tls-assets.service
```

Comment:
`decrypt-tls-assets.service` is AWS specific, right? This does not work that way when we want to have the cloud config manager being abstracted. The dependency statements probably can be the other way around so that we do not pollute the cloud config.